qa/integration-tests/src/test/java/io/zeebe/broker/it/health/HealthMonitoringTest.java

```diff
@@ -77,4 +81,21 @@ private void corruptAllSnapshots(final Broker leader) {
                   .forEach(file -> file.delete());
```

Comment:
We should use the method reference syntax wherever possible

```suggestion
                  .forEach(File::delete);
```